src/Schema/Events/Click.ts

```diff
@@ -176,10 +177,13 @@ export interface ClickedArtistSeriesGroup extends ClickedEntityGroup {
  *    destination_page_owner_slug: "romain-jacquet-lagreze-makeshift-garden-hong-kong",
  *    horizontal_slide_position: 1,
  *    type: "thumbnail"
+ *    signal_labels: ["Limited-Time Offer"],
  *  }
  * ```
  */
-export interface ClickedArtworkGroup extends ClickedEntityGroup {
+export interface ClickedArtworkGroup
+  extends ClickedEntityGroup,
+    CollectorSignalsValues {
```

Comment:
No cleverness in this codebase! Don't want to extend or anything like that, especially in regards to mixing types. Keep this as boring as possible 🙏